Robots.js

```diff
@@ -422,16 +427,51 @@ Robots.prototype.getMatchingLineNumber = function (url, ua) {
 };
 
 /**
- * Returns the opposite of isAllowed()
- *
+ * In standard mode, it returns the opposite of is allowed().
+ * In explicit mode, it will return:
+ *  - true if the the agent is explicitly disallowed (wildcard non included),
+ * 	- throws an error if the user agent is not specified,
+ * 	- and false otherwise.
  * @param  {string}  url
  * @param  {string}  ua
  * @return {boolean}
  */
-Robots.prototype.isDisallowed = function (url, ua) {
-	return !this.isAllowed(url, ua);
+Robots.prototype.isDisallowed = function (url, ua, explicit) {
+	if ((explicit === true) && (ua === undefined)) {
+		throw new Error("User Agent must be specified in explicit mode")
+	}
+
+	var rule = this._getRule(url, ua, explicit);
+	if (typeof rule === 'undefined') {
+		return true;
+	}
+	return !(!rule || rule.allow);
 };
 
+Robots.prototype.isExplicitlyDisallowed = function(url, ua) {	
```

Comment:
This seems to be re-implementing `_getRule()`, is there a reason it doesn't call `_getRule()` with `explicit` set to `true`?

Thinking about it, it might be worth adding an `isExplicitlyAllowed()` method that duplicates `isAllowed()` but uses the explicit rule, something like:
```js
Robots.prototype.isExplicitlyAllowed = function (url, ua) {
        // Not sure if we want to allow the '*' user agent here or not?
	if (ua === undefined) {
		throw new Error("User Agent must be specified in explicit mode")
	}

	var rule = this._getRule(url, ua, true);
	[...]
```
then `isExplicitlyDisallowed()` could negate that like `isDisallowed()` currently does with `isAllowed()`.

That would make the API the same between both versions and avoids any incompatibility risks. What do you think?